stores/filestore.go

```diff
@@ -2827,7 +2827,17 @@ func (ms *FileMsgStore) expireMsgs(now, maxAge int64) int64 {
 				}
 			}
 			if slice != nil {
-				m = ms.getMsgIndex(slice, ms.first)
+				var err error
+				m, err = ms.getMsgIndex(slice, ms.first)
```

Comment:
I think could be one line since not assigning. `if m, err = ..; err != nil {`

Reply:
ok